src/InsertionsClient.Console/ConsoleApp/InputLoading.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Net.Insertions.ConsoleApp
+{
+    internal static class InputLoading
+    {
+        internal static void ProcessArgument(string argument, string appSwitch, string cmdLineMessage, out string target)
```

Comment:
minor: why use of "out" instead of returning a string explicitly in the signature?  The same comment for `ProcessArgumentInt`.

Reply:
these methods almost feel like Try-pattern candidates